odootil/__manifest__.py

```diff
@@ -2,12 +2,12 @@
 # See LICENSE file for full copyright and licensing details.
 {
     'name': "Odoo Utilities",
-    'version': '14.0.1.0.0',
+    'version': '14.0.3.0.0-rc.1',
```

Comment:
Shouldn't it be
```suggestion
    'version': '14.0.2.0.0-rc.1',
```
?

Reply:
No, first merge had incorrect version bump, so it now looks as if its incorrect. Original version is `12.0.2.21.0`, so major version should become 3.